Add Vantage Integration (#1535)
DataDog/integrations-extras: vantage/README.md

# Vantage

## Overview

Vantage is a cloud cost transparency and optimization platform. This integration allows Datadog users to import their Datadog costs into Vantage and track alongside their other infrastructure spend, such as AWS, Snowflake, and Kubernetes.

## Setup

### Installation

Visit [Vantage](https://console.vantage.sh) to sign up for free. Once registered visit the integrations page and add a Datadog integration. This will guide you through the Datadog OAUTH2 flow in order to grant Vantage access to your billing and usage data.

> ## Configuration

Reviewer: ```suggestion
### Configuration
```

Once integrated you can begin exploring your Datadog costs within Vantage. You can create filters for specific Datadog organizations and services alongside costs from any of the other supported Vantage providers.

> ## Troubleshooting

Reviewer: ```suggestion
## Support
```

We only support a couple of specific h2 headers - support makes more sense here

# Vantage

## Overview

> Vantage is a cloud cost transparency and optimization platform. This integration allows Datadog users to import their Datadog costs into Vantage and track alongside their other infrastructure spend, such as AWS, Snowflake, and Kubernetes.

Reviewer 2: Just following up on this -- although `spend` as a noun is becoming more commonly used, it's not universally recognized in all dictionaries, such as Merriam-Webster for example:
https://www.merriam-webster.com/dictionary/spend

As such, this suggestion is focused around accessibility, as some users may not be familiar with this usage yet.

> ### Uninstallation

Reviewer 3: What do you think about moving the `Uninstallation` section closer to the bottom? After `Configuration` and before `Support`. I think the doc will flow better with the `Installation` section up top.

Reviewer: I'm good with this suggestion - having this show up under the Configure tab makes more sense

Reviewer 2: From a documentation perspective, I agree that it makes more sense to relocate the `Uninstallation` section as mentioned here.

> To remove the Datadog integration from Vantage visit the [integrations page](https://console.vantage.sh/settings/integrations) and click remove.

Reviewer 2: ```suggestion
To remove the Datadog integration from Vantage, visit the [integrations page][1] and click **remove**.
```

To ensure this integration is fully uninstalled, be sure to revoke authorization in the Configure tab of this tile or in [Organization Settings > OAuth Apps](https://app.datadoghq.com/organization-settings/oauth-applications). 

Additionally, ensure that all API keys associated with this integration have been disabled by searching for the integration name on the [API Keys management page](https://app.datadoghq.com/organization-settings/api-keys).

## Setup

### Installation

Visit [Vantage](https://console.vantage.sh) to sign up for free. Once registered visit the integrations page and add a Datadog integration. This will guide you through the Datadog OAUTH2 flow in order to grant Vantage access to your billing and usage data.

### Configuration

> Once integrated you can begin exploring your Datadog costs within Vantage. You can create filters for specific Datadog organizations and services alongside costs from any of the other supported Vantage providers.

Reviewer 2: ```suggestion
Once integrated, begin exploring your Datadog costs within Vantage. You can create filters for specific Datadog organizations and services alongside costs from any of the other supported Vantage providers.
```

# Vantage

## Overview

Vantage is a cloud cost transparency and optimization platform. This integration allows Datadog users to import their Datadog costs into Vantage and track them alongside their other infrastructure spending, such as AWS, Snowflake, and Kubernetes.

## Setup

### Installation

> Visit [Vantage][4] to sign up for free. Once registered, visit the Vantage integrations page and add a Datadog integration. This guides you through the Datadog OAUTH2 flow to grant Vantage access to your billing and usage data.

Reviewer: ```suggestion
Visit [Vantage][4] to sign up for free. Once registered, visit the [Vantage integrations page][1] and add a Datadog integration. This guides you through the Datadog OAUTH2 flow to grant Vantage access to your billing and usage data.
```

### Configuration

Once integrated, begin exploring your Datadog costs within Vantage. You can create filters for specific Datadog organizations and services alongside costs from any of the other supported Vantage providers.

### Uninstallation

> To remove the Datadog integration from Vantage, visit the [integrations page][1] and click **remove**.

Reviewer: ```suggestion
To remove the Datadog integration from Vantage, visit the [Vantage integrations page][1] and click **remove**. Additionally, you'll want to uninstall this integration from Datadog by clicking the **Uninstall Integration** button below.
```
